setup_env.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+clear
+dirpath=$(dirname "$(readlink -f "$0")")
+
+# Check if git is installed
+if ! command -v git &> /dev/null; then
+    echo "No git executable found in PATH!"
+    echo
+    read -p "Press any key to continue..."
+    exit 1
+fi
+
+# Install required system packages
```

Comment:
In my opinion, a script like this should not be trying to install system packages. 

I understand the need of making things a bit user-friendlier, but this solution only really covers Debian-based distros that use `apt` to install packages. And even if we tried to support every distro out there, in my opinion we would be over-complicating things for very little gain.

Instead, we should document what system dependencies are necessary to run the script (later in the Wiki), and simply let the script fail when they're are not installed.

Reply:
yee didn't think about that, i'll remove it